src/components/ArticleBody/ArticleBody.tsx

```diff
@@ -75,37 +66,32 @@ const Wrapper = styled.section`
       font-size: ${theme.fonts.sizes.xmedium};
       margin-bottom: ${theme.spacings.small};
     }
-  `}
-`
 
-const CodeHighlighter = styled(SyntaxHighlighter)`
-  ${({ theme }) => css`
-    border-radius: ${theme.spacings.small};
+    .hljs {
+      border-radius: ${theme.spacings.small};
+    }
   `}
 `
 
-const Code = ({ children }: CodeProps) => {
-  const defaultLanguage = 'javascript'
-  return (
-    <CodeHighlighter
-      style={nightOwl}
-      language={defaultLanguage}
-      showLineNumbers
-      wrapLongLines
-    >
-      {String(children)}
-    </CodeHighlighter>
-  )
+const rehypeSanitizeSettings: RehypeSanitizeOptions = {
+  ...defaultSchema,
+  attributes: {
+    ...defaultSchema.attributes,
+    code: [
+      ...((defaultSchema.attributes && defaultSchema.attributes.code) || []),
```

Comment:
Separar em uma const/var separada e com um nome declarativo.

Reply:
What do you think about [this](https://github.com/joaooliveiradev/conduit-frontend/pull/43/commits/7fa55330498ce6c5f76f83be4bf6582beabe7158)?